src/models/People.js

```diff
@@ -297,6 +297,30 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
     return false;
       
   }
+
+  // This method is specifically for the user management page. The admin
+  // can update a user's password without needing their old password,
+  // or if the user has forgotten their password and wants to change it
+  static async resetPassword({ id, newPassword, forcePasswordChange }) {
```

Comment:
This is a pretty powerful method. Let's keep an eye on this to avoid security issues.